lib/ui/views/projects/project_details_view.dart

```diff
@@ -319,9 +326,11 @@ class _ProjectDetailsViewState extends State<ProjectDetailsView> {
                   child: Text('CANCEL'),
                   onPressed: () => Navigator.pop(context),
                 ),
-                FlatButton(
-                  child: Text('ADD'),
-                  onPressed: () => onAddCollaboratorsPressed(context),
+                CVFlatButton(
+                  triggerFunction: onAddCollaboratorsPressed,
+                  context: context,
+                  buttonText: 'ADD',
+                  key: addButtonGlobalKey,
```

Comment:
same as above

Reply:
@Nitish145 Check for requested changes in latest commits.